internal/archive/archive.go

```diff
@@ -290,8 +290,10 @@ func finalizeHeader(header *tar.Header, uid, gid int, mode int64) {
 // 	- UID
 // 	- User Name
 // 	- Group Name
-func NormalizeHeader(header *tar.Header) {
-	header.ModTime = NormalizedDateTime
+func NormalizeHeader(header *tar.Header, normalizeModTime bool) {
+	if normalizeModTime {
+		header.ModTime = NormalizedDateTime
+	}
 	header.Uid = 0
```

Comment:
Curious if these are a concern too. Thoughts?

Reply:
Not necessarily, this is dependent on where the method is called. For example, in [finalizeHeader](https://github.com/buildpacks/pack/pull/505/files#diff-b44fe55937a30f76962a36ee6e048c95R277-R282), it would set the UID and GID to that of the builder/stack so it should be fine.

In [buildpacks.go](https://github.com/buildpacks/pack/pull/505/files#diff-cf43096d3797f1197a38502c90e7af8eR154), we do want UID and GID to be set to zero. For buildpacks, they become root-owned to effectively make them read-only.